app/model/recipes/recipes.go

```diff
@@ -44,3 +49,44 @@ func (r *recipes) AddPhoto(photoPath string, recipeId uuid.UUID, stage string, s
 func (r *recipes) AddStep(recipeId uuid.UUID, stage string, description string) (*model.Recipe, error) {
 	return nil, nil
 }
+
+func (r *recipes) AddTags(recipeId uuid.UUID, tags []string) (*model.Recipe, error) {
+    return nil, nil
+}
+
+func (r *recipes) SetCalories(recipeId uuid.UUID, calories int) (*model.Recipe, error) {
+    return nil, nil
+}
+
+func (r *recipes) CategorizeByCalories(recipes []model.Recipe) (map[string][]model.Recipe, error) {
```

Comment:
we'll need to move this logic somewhere else.
kurwa dopiero się zczaiłem że można po polsku.

to ten, fasada to nie odpowiednie miejsce, póki co niech zostanie, ale fasada jest takim punktem wejściowym tylko do systemu, inne komponenty powinny to handlować czyli jakieś bardziej domenowe serwisiki. Tzn tutaj w tym projekcie to bez sensu ale na tyle często piszę beznadziejny kod zawodowo że chciałbym zadbać o jakość i poonanizować się tutaj :D W każdym razie co do treści nie mam większych zastrzeżeń. Tak to się w golangu robi, nice